v2/azure-arm.yaml

```diff
@@ -871,6 +871,17 @@ objectModelConfiguration:
       PrivateLinkResource:
         Id:
           $armReference: true
+  datafactory:
+    2018-06-01:
```

Comment:
I'm surprised to see ADF haven't released a new version since this one.

Reply:
yea, according to [auto-generated ARM template JSON schema](https://schema.management.azure.com/schemas/common/autogeneratedResources.json), 2018 is the latest api version.